app/src/main/java/com/example/remindmelater/MainActivity.kt

```diff
@@ -48,19 +49,24 @@ class MainActivity : AppCompatActivity(), OnMapReadyCallback {
 
         // Add a marker in Sydney and move the camera
         val sydney = LatLng(-34.0, 151.0)
-        mMap.addMarker(MarkerOptions().position(sydney).title("Marker in Sydney"))
+        mMap.addMarker(MarkerOptions().position(sydney).title(getString(R.string.marker_in_sydney)))
         mMap.moveCamera(CameraUpdateFactory.newLatLng(sydney))
     }
 
 
     @Composable
-    fun MainScreen(name: String) {
+    fun MainScreen() {
         val context = LocalContext.current
+        val clicked = stringResource(R.string.clicked)
+        val remindMeLater = stringResource(R.string.remind_me_later)
+        val reminderList = stringResource(R.string.reminder_list)
+        val mapViewLabel = stringResource(R.string.map_view_label)
+        val others = stringResource(R.string.others)
```

Comment:
I tried to key property names as close to the text as possible but made deviations based on the likelihood of a variable collision.

Reply:
You could declare these where used... which would obviate the need for storing them in a variable.
One question: is this test data?  If so, it really belongs in a stub/mock class anyway; not hardcoded in the UI.  In that case, no need to extract string resources.